modules/openapi-generator/src/main/resources/go-server/partial_header.mustache

```diff
@@ -1,3 +1,5 @@
+// Code generated by OpenAPI Generator (https://openapi-generator.tech); DO NOT EDIT.
+
 /*
```

Comment:
i'll file another PR to move it inside the /* .. */ comment block below to make the comment a bit more consistent.

Reply:
filed https://github.com/OpenAPITools/openapi-generator/pull/18319